modules/installation/pages/install-proxy.adoc

```diff
@@ -33,6 +42,35 @@ The primary goal of a {susemgrproxy} is to improve {productname} performance by
 
 
 
+.Procedure: Installing {susemgr} Proxy from a DVD image
+
+. Boot your proxy server from the installation image, you might need to adjust the boot order in the BIOS.
+. When prompted, select [guimenu]``Installation``.
++
+image::unifiedinstaller-install.png[scaledwidth=80%]
+. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check the [guimenu]``SUSE Manager Proxy`` checkbox, and click btn:[Next].
++
+image::unifiedinstaller-proxy.png[scaledwidth=80%]
+. Read and agree to the End User Licence Agreement, and click btn:[Next].
+. In the [guimenu]``Registration`` screen, check the [guimenu]``Register System via scc.suse.com`` checkbox, enter your {scc} credentials, and click btn:[Next].
+. OPTIONAL: In the [guimenu]``Add On Product`` screen, select any additional or add-on products you require, and click btn:[Next].
+. In the [guimenu]``System Role`` screen, check the [guimenu]``SUSE Manager Proxy`` checkbox, and click btn:[Next].
+. In the [guimenu]``Suggested Partitioning`` screen, accept the default values, or use the btn:[Guided Setup] or btn:[Expert Practitioner] options to customize your partitioning model, and click btn:[Next].
+. In the [guimenu]``Clock and Time Zone`` screen, enter your region and timezone, and click btn:[Next].
+. In the [guimenu]``Local Users`` screen, create a new user, and click btn:[Next].
+. Review the settings on the [guimenu]``Installation Settings`` screen, and once you are happy click btn:[Install].
+
```

Comment:
Installing from unified installer directly is an option, but not the prefered one.
We need to describe how to install a proxy from suse manager server using autoyast.

This requires an example autoyast profile which should include also sections about partitioning.

But this section is still usefull as of now this is the only way to install a proxy because autoinstallation does not work yet.

@mantel - maybe you can work on an example profile?

Reply:
@mantel Anything you can provide would be helpful!